src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt

```diff
@@ -39,68 +36,58 @@ class ContextFilesPanel(
     updateContentWith(chatMessage.contextFiles)
   }
 
-  @RequiresBackgroundThread
-  private fun updateFileList(contextFileNames: Set<String>) {
-    val filesAvailableInEditor =
-        contextFileNames
-            .map(Paths::get)
-            .mapNotNull(VirtualFileManager.getInstance()::findFileByNioPath)
-            .toList()
+  fun updateContentWith(contextFiles: List<ContextFile>?) {
+    val contextFileFiles =
+        contextFiles?.mapNotNull { it as? ContextFileFile }?.filter { it.repoName == null }
```

Comment:
the handling for `it.repoName != null` is in the next PR: https://github.com/sourcegraph/jetbrains/pull/559